src/utils/apiUtils/chartData.js

```diff
@@ -22,159 +18,33 @@ import {
-  } = options;
-  return {
-    value,
-    optionalValue,
-    valueFormat,
-    optionalValueFormat
-  };
-}
-
-function processStockData({
-  averageVolume,
-  currentPrice,
-  dayHigh,
-  dayLow,
-  dividendRate,
-  dividendYield,
-  freeCashflow,
-  marketCap,
-  open,
-  previousClose,
-  returnOnEquity,
-  trailingEps,
-  trailingPE,
-  volume
-}) {
-  return {
-    [LABEL_PREVIOUS_CLOSE]: createStockDataEntry(previousClose),
-    [LABEL_CURRENT_PRICE]: createStockDataEntry(currentPrice),
-    [LABEL_OPEN]: createStockDataEntry(open),
-    [LABEL_HIGH]: createStockDataEntry(dayHigh),
-    [LABEL_LOW]: createStockDataEntry(dayLow),
-    [LABEL_DIVIDEND]: createStockDataEntry(
-      dividendRate,
-      {
-        optionalValue: dividendYield,
-        optionalValueFormat: NUMBER_FORMAT_PERCENT
-      }
-    ),
-    [LABEL_MARKET_CAP]: createStockDataEntry(marketCap,
-      {
-        valueFormat: NUMBER_FORMAT_SHORT_SUFFIXED
-      }),
-    [LABEL_VOLUME]: createStockDataEntry(
-      volume,
-      {
-        valueFormat: NUMBER_FORMAT_SHORT_SUFFIXED,
-        optionalValue: averageVolume,
-        optionalValueFormat: NUMBER_FORMAT_SHORT_SUFFIXED
-      }
-    ),
-    [LABEL_PE_RATIO]: createStockDataEntry(
-      trailingPE,
-      {
-        optionalValue: trailingEps
-      }
-    ),
-    [LABEL_ROE]: createStockDataEntry(
-      returnOnEquity,
-      {
-        valueFormat: NUMBER_FORMAT_PERCENT
-      }
-    ),
-    [LABEL_FCFY]: createStockDataEntry(
-      calculateFcfy(freeCashflow, marketCap),
-      {
-        valueFormat: NUMBER_FORMAT_PERCENT
-      }
-    )
-  };
-}
-
-function createStock(stockQuote) {
-  const {
-    price,
-    summaryDetail,
-    financialData,
-    defaultKeyStatistics
-  } = stockQuote;
-  return {
-    companyName: price.shortName,
-    symbol: price.symbol,
-    exchange: price.exchangeName,
-    stockOverviewData: processStockData(
-      Object.assign(
-        {},
-        summaryDetail,
-        financialData,
-        defaultKeyStatistics
-      )
-    )
-  };
-}
-
-// Used for historical() data obtained using period 'd'
-function getDatesAndPrices(dailyData) {
-  let datesAndPrices = [];
-  dailyData.forEach(({
-    date,
-    close
-  }) => {
-    datesAndPrices.unshift({
-      date: formatDateForMaxStockData(date),
-      price: close
-    });
-  });
-  return datesAndPrices;
-}
+import { createStock } from '../stockDataUtils/createStock';
 
 // NOTE: this date has timezone UTC, which is incorrect but works in this
 // case because we are just extracting the time
-function getAdjustedDateForTimestamp(gmtoffset, timestamp) {
+export function getAdjustedDateForTimestamp(gmtoffset, timestamp) {
   const adjustedTimestamp =
     (timestamp + gmtoffset) * MILLISECONDS_PER_SECOND;
   return new Date(adjustedTimestamp);
 }
 
-function getDateAndTime(gmtoffset, timestamp, dateAndTimeFormat) {
+export function getDateAndTime(gmtoffset, timestamp, dateAndTimeFormat) {
```

Comment:
Not really relevant for this PR's changes but this method name doesn't tell me what this function does.  Maybe `getFormattedAdjustedDateForTimestamp`/`formatAndAdjustDateForTimestamp`/`getFormattedLocalizedDateForTimestamp`/`formatAndLocalizeDateForTimestamp`?